desktop/ELECTRON_EVENTS.ts

```diff
@@ -14,6 +14,7 @@ const ELECTRON_EVENTS = {
     DOWNLOAD_FAILED: 'download-started',
     DOWNLOAD_CANCELED: 'download-canceled',
     SILENT_UPDATE: 'silent-update',
+    OPEN_LOCATION_SETTING: 'openLocationSetting',
```

Comment:
All other events here use kebab-case, let's use the same here
```suggestion
    OPEN_LOCATION_SETTING: 'open-location-setting',
```